components/feature/pwa/src/main/java/mozilla/components/feature/pwa/ManifestStorage.kt

```diff
@@ -25,10 +25,24 @@ class ManifestStorage(context: Context) {
      * Load a Web App Manifest for the given URL from disk.
      * If no manifest is found, null is returned.
      *
-     * @param startUrl URL of site. Should correspond to manifest's start_url.
+     * @param url URL of site. Should correspond to manifest's start_url.
      */
-    suspend fun loadManifest(startUrl: String): WebAppManifest? = withContext(Dispatchers.IO) {
-        manifestDao.value.getManifest(startUrl)?.manifest
+    suspend fun loadManifest(
+        url: String,
+        allowTimedout: Boolean = false
+    ): WebAppManifest? = withContext(Dispatchers.IO) {
+        val entry = manifestDao.value.getManifest(url)
+
+        if (entry == null) {
```

Comment:
You can simplify this with the elvis operator
```kt
val entry = manifestDao.value.getManifest(url) ?: return@withContext null
```
Does return without the `@withContext` label work?

Reply:
@NotWoods I always get `'return' is not allowed here` without the label.